CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ This changelog keeps track of all changes to the Packs SDK. We follow convention
 ### Changed
 
 - Updated `PrecannedDateRange` to match the date range picker in the Coda UI: added `Last90Days`, `Last180Days`, `Last365Days`, `Next90Days`, `Next180Days`, `Next365Days`, `Last7AndNext7Days`, `Last30AndNext30Days`, deprecated `Last3Months`, `Last6Months`, `Next3Months`, and `Next6Months`, and removed `ThisWeekStart`, `ThisMonthStart`, and `ThisYearStart`, which never actually worked.
+- Improved `eslint` calls in `Makefile` to ensure the associated ESLint config files are applied as expected.
```

Comment:
We tend to only include changes in the changelog if consumers of the SDK will notice them. A change to the eslint config only affects maintainers, so I'd omit it.

Reply:
OK, good to know -- I'll remove this.